.gitignore

```diff
@@ -1,10 +1,9 @@
 *.pyc
-db.sqlite3
 .coverage
 .pydevproject
 .project
 *eclipse*
 *~
 mobile/platforms
 mobile/plugins
-venv/
\ No newline at end of file
+venv/
```

Comment:
Zakaj se spreminja `.gitignore`?

Reply:
Jaz sem samo počstil PR in ga splittal na dva, ampak ja, to bi pasalo kot posebej commit.

Razbil sem na dva `.gitignore`, da je drugi lokalen znotraj `web` pa potem odstranil nepotrebna pravila, ker jih splošnejša nadomestijo.

Reply:
Aha, torej to je smiselno, da je notri. Ok.